testgrid/config/config.yaml

```diff
@@ -874,6 +874,9 @@ test_groups:
 # azure
 - name: kubernetes-azure
   gcs_prefix: kube_azure_log/kubernetes.azure.nightly
+# Canonical Distribution of Kubernetes (contact: @chuckbutler on github)
+- name: canonical-kubernetes-e2e-gce
+  gcs_prefix: canonical-kubernetes-tests/gce/logs/e2e-node
```

Comment:
I'd expect this path and `buckets.yaml` to match, but they don't. should `buckets.yaml` be updated to `gs://canonical-kubernetes-tests/gce/logs`?